src/pymatgen/io/vasp/MP24RelaxSet.yaml

```diff
@@ -0,0 +1,129 @@
+# Default VASP settings for calculations in the Materials Project
+# using the Regularized-Restored Strongly Constrained and Appropriately
+# Normed functional (r2SCAN).
+PARENT: VASPIncarBase
+INCAR:
+  ALGO: Normal
+  EDIFF: 1.e-05
+  EDIFFG: -0.02
+  ENAUG: 1360
+  ENCUT: 680
+  IBRION: 2
+  ISIF: 3
+  ISMEAR: 0 # included to have some reasonable default
+  ISPIN: 2
+  KSPACING: 0.22 # included to have some reasonable default
+  LAECHG: True
+  LASPH: True
+  LCHARG: True
+  LELF: False # LELF = True restricts calculation to KPAR = 1
```

Comment:
I believe it restricts to NPAR = 1 not KPAR = 1. I am not sure where this restriction originates.

There is some logic that expensive-to-compute quantities should be saved by default? (e.g., to my knowledge the only way to recover the KE density from a VASP calculation is to store the ELF)

I see the sense in leaving this as `False` but I wonder what should be stored by default? I imagine there is a trade-off with I/O and post-processing time too, especially for simple calculations.

Reply:
The [manual says NPAR = 1](https://www.vasp.at/wiki/index.php/LELF), but the actual error message I get when I run VASP 6.4.2 (just checked) is: `ELF: KPAR>1 not implemented, sorry.` Either way, restricts the parallelization

You can reconstruct the KED from the orbitals in WAVECAR, since those should just be the plane wave coefficients + wave vectors (think we have tools in PMG to parse this but haven't looked thoroughly)

I'm good with `LELF = True` for the final static in the MP workflow, since this is usually very cheap (it starts from the WAVECAR of the r2SCAN relaxation)

Reply:
So we hard-code `KPAR: 1` for the final static?

Makes sense with the WAVECAR, I am genuinely wondering whether one day we should encourage storage of this by default if storage costs continue to reduce. I am encountering more and more post-processing tools that would really benefit if a large dataset of wave functions were available. I wonder how compressible they are with modern methods too, something I will not have time to investigate unfortunately!

Reply:
A little hesitant to do that, especially if we want to treat larger systems, but it'll require some testing for me to see if `KPAR = 1` is a real practical issue

Saving the [vaspwave.h5 file](https://www.vasp.at/wiki/index.php/Vaspwave.h5) might be better in the long run. There's currently an [open PR](https://github.com/materialsproject/pymatgen/pull/3768) for this, I also need to finish the vaspout.h5 PR